tests/prefetch_related/tests.py

```diff
@@ -1999,6 +1999,21 @@ def test_window_not_supported(self):
         with self.assertRaisesMessage(NotSupportedError, msg):
             list(Book.objects.prefetch_related(Prefetch("authors", authors[1:])))
 
+    @skipUnlessDBFeature("supports_over_clause")
+    def test_empty_order(self):
+        authors = Author.objects.order_by()
+        with self.assertNumQueries(3):
+            books = list(
+                Book.objects.prefetch_related(
+                    Prefetch("authors", authors),
+                    Prefetch("authors", authors[:1], to_attr="authors_sliced"),
```

Comment:
Unfortunately, this crashes on Oracle: ```ORA-30485: missing ORDER BY expression in the window specification```.

Reply:
Any thoughts about adding an explicit `order_by=("pk",)` if the backend requires one with a feature flag?

Reply:
`ORDER BY` is required for `ROW_NUMBER()`, see [docs](https://docs.oracle.com/en/database/oracle/oracle-database/21/sqlrf/ROW_NUMBER.html#GUID-D5A157F8-0F53-45BD-BF8C-AE79B1DB8C41).

Reply:
We have this noted in a docstring :smiling_face_with_tear: 
https://github.com/django/django/blob/ca47884410acb7cb1d36bc8a573b67a5b197d3e2/tests/expressions_window/tests.py#L204-L210

Reply:
I think we're saying the same thing, let me push a commit to demonstrate what I had in mind.

Reply:
> Any thoughts about adding an explicit `order_by=("pk",)` if the backend requires one based on a feature flag?

For now, I would skip it like we do for `test_row_number_no_ordering`:

https://github.com/django/django/blob/ca47884410acb7cb1d36bc8a573b67a5b197d3e2/django/db/backends/oracle/features.py#L116-L119

Reply:
Just pushed a commit to this effect.